src/index.js

```diff
@@ -182,10 +183,18 @@ function addMetadata({ types: t }) {
       CallExpression(babelPath, state) {
         if (!state.opts.shouldLoadFile) return;
 
+        let moduleArgs;
+
         // If this call expression is a top-level module, store string arg, else skip the nested module entirely
         if (babelPath.get('callee').isIdentifier({ name: 'module' })) {
           if (babelPath.parentPath.parent.type === 'Program') {
-            state.opts.moduleName = babelPath.get('arguments')[0].node.value;
+            moduleArgs = babelPath.get('arguments');
+
+            const moduleFunction = moduleArgs[1];
```

Comment:
I feel like you could probably destructure the array above with this and line 197 below.